### ISSUE TITLE
Umbraco Cloud ID login fails when adding appsettings in location element in web.config

### ISSUE THREAD

**Issue by PGGMTeam5** [umbraco/Umbraco.Cloud.Issues#574]
### Issue description

Using Umbraco Cloud project v8.18.3

We have the need to add some specific appsettings for the Umbraco backoffice (related to CSP headers).
One way of doing this, is to add these appsettings to the (already existing) <location path="umbraco"> element in the web.config.

However, even just adding a test setting like this:
```
<location path="umbraco">
 <appSettings>
     <add key="foo" value="bar" />
</appSettings>
</location>

```
will then break the Umbraco Cloud login.

Using the link "Go to backoffice" in an incognito window will give the Umbraco Cloud ID login view.
After entering the correct login credentials, the following error is displayed:
```
Login failed
AADB2C90088: The provided grant has not been issued for this endpoint. Actual Value : B2C_1A_SignInLocal and Expected Value : B2C_1A_SignInBackOffice

```

**Comment by sajumb** [comment]
Thanks for reaching out. I have brought your question to the attention of our cloud team. I will post the response I get here.

**Comment by sajumb** [comment]
I have talked to our 2nd level support who referenced the support ticket that led to this feature request. I have created a ticket in our backlog for this request.

**Comment by jesp209i** [comment]
this issue was resolved according to #573 
Closing this issue :)
